### PR TITLE
Update id.m3u

### PR DESCRIPTION
**Pull request by azelfafa**
+ RCTI, MNC TV, and GTV

### CODE REVIEW

File: channels/id.m3u [hunk 1]
```diff
@@ -37,6 +37,12 @@ https://live-edge01.telecentro.net.ar/live/smil:musictop.smil/playlist.m3u8
 https://59d7d6f47d7fc.streamlock.net/canale51/canale51/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="ID" tvg-language="Indonesian" tvg-logo="http://image.azuza.web.id/images/agus-1545452020.jpg" group-title="Religious",Reformed 21
 http://edge.linknetott.swiftserve.com/live/BsNew/amlst:reformedch/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="ID" tvg-language="Indonesian" tvg-logo="https://i.postimg.cc/13Nm4VLb/20191010-164600.png" group-title="",RCTI
+https://iptv-ppvhxd1r4.vercel.app/api/vidio?id=665
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="ID" tvg-language="Indonesian" tvg-logo="https://i.postimg.cc/rw8620qS/20191026-003920.png" group-title="",MNCTV
+https://iptv-ppvhxd1r4.vercel.app/api/vidio?id=870
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="ID" tvg-language="Indonesian" tvg-logo="https://i.postimg.cc/HsqNM8Hc/globaltv.png" group-title="",GTV
+https://iptv-ppvhxd1r4.vercel.app/api/vidio?id=778
```
Comment (freearhey) [inline channels/id.m3u:45]: None of the links work.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: channels/id.m3u (hunk 1 above)
Comment (Dzaky05) [reply channels/id.m3u]: I was fixed the link.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
@Dzaky05 Thank you for your help!

**Pull request merged**